x/ibc/light-clients/07-tendermint/types/update.go

```diff
@@ -178,5 +178,9 @@ func update(clientState *ClientState, header *Header) (*ClientState, *ConsensusS
 		NextValidatorsHash: header.Header.NextValidatorsHash,
 	}
 
+	// set context time as processed time as this is state internal to tendermint client logic.
+	// client state and consensus state will be set by client keeper
+	SetProcessedTime(clientStore, header.GetHeight(), uint64(ctx.BlockTime().UnixNano()))
```

Comment:
We need to do this in `upgrade.go`, right?

Reply:
given that this was not caught by tests, we should add a test that sends a packet, recvs, acks, all after the upgrade. That test would have caught the missing processed time

Reply:
No, we don't do this because we don't want any packet verification to occur against the upgrade consensus state. Note that the root is empty. The next submitted consensus state can be used for packet verifcation.

Will document this as its non-obvious